### ISSUE TITLE
Flag locations as paused

### ISSUE THREAD

**Issue by nickelswitte** [cupcakearmy/autorestic#379]
**Is your feature request related to a problem? Please describe.**
I currently have one very large location, that I dont want to back up at the moment. But I would like to keep it in the .autorestic.yml file with all the hooks and configuration there. I just did `autorestic backup -a` and it (of course) backed up this large location as well. 

**Describe the solution you'd like**
It would be nice to have a flag like `status: paused`, which would make autorestic skip this location for all future backups, until I set it back to `status: active`.

**Comment by guest20** [comment]
Can't you comment the entry out in the yaml file? 

Another option is to version your config, delete the entry, commit it and later, when you want it back you can `git revert that-commit-id`

**Comment by nickelswitte** [comment]
That is true, both are good solutions. However, a builtin `status` field would still be the cleanest option in my opinion.
